ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts

```diff
@@ -325,7 +325,11 @@ class PlotLineOrBand {
                     align: optionsLabel.textAlign || optionsLabel.align,
                     rotation: optionsLabel.rotation,
                     'class': 'highcharts-plot-' + (isBand ? 'band' : 'line') +
-                        '-label ' + (optionsLabel.className || ''),
+                        '-label' + (
+                        optionsLabel.className ?
+                            '-' + optionsLabel.className :
```

Comment:
This can't be right? So if I set the `className` to `my-label`, the output will be `highcharts-plot-band-label-my-label`? That's not the intension...

Reply:
Ah, you're right. There should be a space and that change was applied before https://github.com/highcharts/highcharts/pull/20708. I'll remove my changes.